src/edu/ucsb/nceas/metacat/dataone/v1/CNodeService.java

```diff
@@ -107,7 +107,7 @@ public static CNodeService getInstance(HttpServletRequest request) {
 	 */
 	private CNodeService(HttpServletRequest request) {
 
-		logMetacat = Logger.getLogger(CNodeService.class);
+		logMetacat = LogFactory.getLog(CNodeService.class);
```

Comment:
```suggestion
```

Reply:
See the commit 70db73cab0ce506277782cd20f081b0dabcd9c8a.